taipy/core/taipy.py

```diff
@@ -420,6 +420,23 @@ def get_scenarios(cycle: Optional[Cycle] = None, tag: Optional[str] = None) -> L
     return []
 
 
+def get_scenarios_by_time( Year: int , Month: int , Day: int, Hour: int , Minute : int ) -> List[Scenario]:
```

Comment:
```suggestion
def get_scenarios_by_time(year: int , month: int , day: int, hour: int , minute : int ) -> List[Scenario]:
```

The parameters should be on lower case. And I believe the method should not require all of these parameters to work.